### PR TITLE
feat(route): 增加bilibili合集的订阅

### PR DESCRIPTION
**Pull request by shinonomeow**
## 该 PR 相关 Issue / Involved issue

Open #10162

## 完整路由地址 / Example for the proposed route(s)

```routes
/bilibili/collection/245645656/529166
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: docs/social-media.md [hunk 1]
```diff
@@ -340,6 +340,10 @@ Tiny Tiny RSS 会给所有 iframe 元素添加 `sandbox="allow-scripts"` 属性
 
 </Route>
 
+### 合集更新
+
+<Route author="shininome" example="/bilibili/collection/245645656/529166" path="/bilibili/collection/:uid/:sid/:disableEmbed?" :paramsDesc="['用户 id, 可在 UP 主主页中找到','合集的id,合集上面有','默认为开启内嵌视频, 任意值为关闭']"/>
```
Comment (TonyRL) [inline docs/social-media.md:345]:
```suggestion
<Route author="shininome" example="/bilibili/collection/245645656/529166" path="/bilibili/collection/:uid/:sid/:disableEmbed?" :paramsDesc="['用户 id，可在 UP 主主页中找到', '合集的 id，合集上面有', '默认为开启内嵌视频，任意值为关闭']"/>
```

File: lib/v2/bilibili/collection.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        headers: {
+            Referer: `https://space.bilibili.com/${uid}/channel/collectiondetail?sid=${sid}`,
+        },
+    });
+    const data = response.data;
+
+    ctx.state.data = {
+        title: `${name} 的 bilibili 空间`,
+        link: `https://space.bilibili.com/${uid}/channel/collectiondetail?sid=${sid}`,
+        description: `${name} 的 bilibili 专栏`,
```
Comment (TonyRL) [inline lib/v2/bilibili/collection.js:23]: `title` and `description` are wrong

File: lib/v2/bilibili/router.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 module.exports = (router) => {
+    router.get('/collection/:uid/:sid/:disableEmbed?', require('./collection'));
```
Comment (TonyRL) [inline lib/v2/bilibili/router.js:2]: Please sort it in alphabetical order

**Review by TonyRL — CHANGES_REQUESTED** [review]
Please update `maintainer.js` and `radar.js`

File: lib/v2/bilibili/collection.js (hunk 2 above)
Comment (shininome) [reply lib/v2/bilibili/collection.js:23]: 嘿嘿，偷懒了，这就改

**Pull request merged**